src/test/java/TestPhantomJS.java

```diff
@@ -65,7 +89,60 @@ public void testFooterAndHeader() throws IOException, RenderException {
 
         try (InputStream is = PhantomJS.render(TestPhantomJS.class.getResourceAsStream("test.html"), RenderOptions.DEFAULT.withHeaderInfo(header).withFooterInfo(footer))) {
             PDDocument doc = PDDocument.load(is);
-            assertTrue(doc.getNumberOfPages() == 2);
+            doc.close();
+            assertTrue(doc.getNumberOfPages() == 2); //Only 1 page is generated on Windows. This seems platform dependent.
```

Comment:
@moodysalem Left the tested value to 2 in order to satisfy Travis CI tests. Will look into possibly tweaking this setting to get consistent results on Windows.